packages/wpcom-template-parts/src/universal-footer-navigation/index.tsx

```diff
@@ -131,7 +131,7 @@ export const PureUniversalNavbarFooter = ( {
 								</li>
 								<li>
 									<a
-										href={ localizeUrl( 'https://wordpress.com/do-it-for-me/?ref=footer_pricing' ) }
+										href={ localizeUrl( 'https://wordpress.com/built-by/?ref=footer_pricing' ) }
```

Comment:
cc @arcangelini on this change. I haven't figured out how to test this yet, but it's such a minor change that I think a visual check is fine.

Reply:
You should find this footer on /themes and /plugins while logged out.
I tested and it looks good 👍